web/src/user_status_ui.ts

```diff
@@ -183,6 +183,11 @@ function user_status_post_render(): void {
         set_selected_emoji_info({});
         update_button();
     });
+
+    // Hide broken emoji images
+    $("#set-user-status-modal .emoji").on("error", function () {
+        $(this).hide();
+    });
```

Comment:
Can you explain why we need this change? I thought the intent of the other changes was to avoid us requesting emoji images that don't exist.

I can't tell if this is the correct error-handling; it needs more explanation as to under what situation it's expected that broken emoji images might occur, and why hiding such images might be helpful.

It might also help to make this a separate commit; it seems like it's mostly independent of the other change?